.readthedocs.yml

```diff
@@ -14,4 +14,4 @@ python:
 
 sphinx:
   builder: html
-  fail_on_warning: true
+  fail_on_warning: false
```

Comment:
I feel this really needs to be fixed before it can be merged; what is the warning?

Reply:
Fixed! It was a very silly thing - a title underline wasn't as long as the updated title. I was just doing a bad job of reading the readthedocs CI output.